FundacionParaguaya/MentorApp
# Replacing picker with a custom select

There are many issues surrounding the native Picker element, so we are replacing it with a custom element.

**To Test**
A select now should open a modal from the bottom of the screen with the options, including an empty `--` one. Otherwise the select should handle the same way. Go through a whole life map and check if all select element are behaving properly. Make sure to save some data and come back to see if the draft has saved it properly

**Related issues**
- Resolves #141
- Resolves #119
- Resolves #174
